bbq/vm/config.go

```diff
@@ -329,4 +371,61 @@ func (c *Config) SetAttachmentIteration(composite *interpreter.CompositeValue, s
 	return false
 }
 
+func (c *Config) RecoverErrors(onError func(error)) {
```

Comment:
These and below functions are just to make things work. We need to properly modularize them later.